extensions/adobe/experience/adcloud/searchads/adgroup.schema.json

```diff
@@ -80,7 +80,7 @@
             "Dummy": "Dummy"
```

Comment:
What does status `Dummy` mean?

Reply:
This is one of the possible status defined in the source database (PG).  It is possible that none of the ad-groups may have that status but application logic allows to have that status.